_includes/config.js.md

```diff
@@ -0,0 +1,413 @@
+
+This value is set to `"default"` by default.
+
+### `prefetch`
+
+When set to `true`, The Lounge will load thumbnails and site descriptions
+from URLs posted in channels and private messages.
+
+This value is set to `false` by default.
+
+### `prefetchStorage`
+
+When set to `true`, The Lounge will store and proxy prefetched images and
+thumbnails on the filesystem rather than directly display the content at
+the original URLs.
+
+This improves security and privacy by not exposing the client IP address,
+always loading images from The Lounge and making all assets secure, which
+resolves mixed content warnings.
+
+If storage is enabled, The Lounge will fetch and store images and thumbnails
+in the `${THELOUNGE_HOME}/storage` folder.
+
+Images are deleted when they are no longer referenced by any message
+(controlled by `maxHistory`), and the folder is cleaned up when The Lounge
+restarts.
+
+This value is set to `false` by default.
+
+### `prefetchMaxImageSize`
+
+When `prefetch` is enabled, images will only be displayed if their file
+size does not exceed this limit.
+
+This value is set to `2048` kilobytes by default.
+
+### `fileUpload`
+
+Allow uploading files to the server hosting The Lounge.
+
+Files are stored in the `${THELOUNGE_HOME}/uploads` folder, do not expire,
+and are not removed by The Lounge. This may cause issues depending on your
+hardware, for example in terms of disk usage.
+
+The available keys for the `fileUpload` object are:
+
+- `enable`: When set to `true`, files can be uploaded on the client with a
+  drag-and-drop or using the upload dialog.
+- `maxFileSize`: When file upload is enabled, users sending files above
+  this limit will be prompted an error message in their browser. A value of
+  `-1` disables the file size limit and allows files of any size. **Use at
+  your own risk.** This value is set to `10240` kilobytes by default.
+
+### `transports`
+
+Set `socket.io` transports.
+
+This value is set to `["polling", "websocket"]` by default.
+
+### `leaveMessage`
+
+Set users' default `quit` and `part` messages if they are not providing
+one.
+
+This value is set to `"The Lounge - https://thelounge.chat"` by
+default.
+
+## Default network
+
+### `defaults`
+
+Specifies default network information that will be used as placeholder
+values in the *Connect* window.
+
+The available keys for the `defaults` object are:
+
+- `name`: Name to display in the channel list of The Lounge. This value is
+  not forwarded to the IRC network.
+- `host`
+- `port`: Usually 6667 for unencrypted connections and 6697 for
+  connections encrypted with TLS.
+- `password`
+- `tls`: Enable TLS connections
+- `rejectUnauthorized`: Whether the server certificate should be verified
+  against the list of supplied Certificate Authorities (CAs) by your
+  Node.js installation.
+- `nick`: Percent signs (`%`) will be replaced by random numbers from 0 to
+  9. For example, `Guest%%%` may become `Guest123`.
+- `username`
+- `realname`
```

Comment:
Some of the fields don't have any descriptions.

Reply:
Yeah, that was on purpose because otherwise it leads to `username: User name`, which is, you know, not much of an improvement I think.